design-system/materials/internals/definition.yaml

```diff
@@ -308,6 +312,8 @@ decisionGroupsByTheme:
           choice: color-surface
         border-color-for-button-as-icon-when-disabled:
           choice: color-neutral
+        border-color-for-card-when-raised:
+          choice: color-neutral-90
```

Comment:
This token is inside the default `theme` group so the value here should be `color-surface` since we don't want the border to be visible in this theme.
Also, you would need to add this token in the `test` group with this value (`color-neutral-90`).